tests/converter/test_image_overlay_converter.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import os
+
+import pytest
+from PIL import Image
+
+from pyrit.prompt_converter import ImageOverlayConverter
+
+from io import BytesIO
+
+
+@pytest.fixture
+def base_image_path():
+    img = Image.new("RGB", (100, 100), color=(255, 255, 255))
+    img.save("base_test.png")
+    return "base_test.png"
+
+
+@pytest.fixture
+def overlay_image_byte():
+    img = Image.new("RGBA", (20, 20), color=(125, 125, 125, 125))
```

Comment:
I looked at your issue and the provided database, but I am still not sure what is the expected implementation here. I think it would be better if a user can manipulate the overlay image alpha (or transparency).